tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -147,8 +149,12 @@
         #end if    
         
         ## view/sort and output BAM file
-        | samtools view -Su - | samtools sort -o - - > $output
-        
+        #if ( str( $analysis_type.analysis_type_selector ) != "full" or str( $analysis_type.sam_opt ) != "true" ):
+          | samtools view -Su - | samtools sort -o - - > $output_1
+        #else
+          > $output_2
+        #end if
+
```

Comment:
Yes, those are the really problematic ones - it looks like we can get away with the greater-than-sign but it would be best practise to escape this.

@jsh58 - this was a pre-existing issue, but since you were working on this bit of the wrapper would you mind adding a commit replacing these with ``&gt;`` please?